modules/services/analytics/src/main/java/au/com/shiftyjelly/pocketcasts/analytics/AnalyticsTracker.kt

```diff
@@ -11,8 +11,8 @@ object AnalyticsTracker {
         trackers.forEach { it.clearAllData() }
```

Comment:
Should we be calling `clearAllData` every time we init the `AnalyticsTracker` (which occurs each time the app starts)? This doesn't effect this PR (because the BumpStatsTracker doesn't do anything when `clearAllData` gets called), but this didn't feel right to me and I was wondering if maybe I was missing something.

Reply:
Good catch! I think this was added when we regenerated anonID every 30 minutes and on application init. We should remove this now.